src/server/index.js

```diff
@@ -66,12 +66,27 @@ router.post('/register', function(req, res) {
   }
 });
 
+router.get('/boards', function(req, res) {
+  let username = jwtVerify(req.headers.token);
+
+  if (!username) {
+    res.status(forbidden).json({message: 'Please login first!'});
```

Comment:
maybe we could use the actual status codes here and just disable eslint for that